test/radius.test.js

```diff
@@ -929,5 +929,22 @@ module.exports = testCase({
     }, decoded.attributes );
 
     test.done();
+  },
+
+  test_invalid_packet_attribute_length: function(test) {
+    var invalid_packet  = fs.readFileSync(__dirname + '/captures/invalid_register.packet');
+    var raw_packet      = fs.readFileSync(__dirname + '/captures/aruba_mac_auth.packet');
+
+    // should fail decode packet attributes
+    test.throws( function() {
```

Comment:
Can you delete the space inside the "throws" parens so it's consistent with the rest of file?